TrackletAlgorithm/Constants.hh

```diff
@@ -19,7 +24,6 @@ constexpr int kNBits_BX = 3;
 constexpr int kNBits_MemAddrBinned = 4;
 constexpr int kMemDepthBinned = (1<<kNBits_MemAddrBinned);
 
-constexpr int kNBits_z = 3;
```

Comment:
Did you get rid of this const on purpose? I think the MatchEngine uses it.

Reply:
Nope not on purpose - sorry! Back now ..